test/e2e/framework/util.go

```diff
@@ -1352,6 +1352,11 @@ func ExpectEqual(actual interface{}, extra interface{}, explain ...interface{})
 	gomega.Expect(actual).To(gomega.Equal(extra), explain...)
 }
 
+// ExpectNotEqual expects the specified two are not the same, otherwise an exception raises
+func ExpectNotEqual(actual interface{}, extra interface{}, explain ...interface{}) {
```

Comment:
I know we are just being consistent with `ExpectEqual` here, but curious why it's called `extra` instead of something like `expected`?